src/mrack/providers/openstack.py

```diff
@@ -117,8 +121,102 @@ async def _openstack_gather_responses(self, *calls):
 
         return result
 
+    async def _create_session(self):
+        """
+        Create session object using credentials.
+
+        Credentials are retrieved from either enviromental variables or
+        clouds.yaml file.
+        Both username+password and application credentials formats
+        are accepted.
+
+        Returns: Session object.
+        """
+        try:
+            # Create session from environment variables
+            return AuthPassword()
+        except TypeError:
+            # Fallback to create session from clouds.yaml file
+            try:
+                config = os_client_config.OpenStackConfig()
+
+                # Get the cloud specified in provisioning-config
```

Comment:
Here is a question. Should `OS_CLOUD` override `profile` from provisioning config?  

The use case is to pick with what credentials to authenticate in LTE. As users might have different name or preferred cloud than the one in provisioning config. And current provisioning config wins. 

It is not that easy to pick the right cloud by changing the value provisioning config.

E.g. if we do in, e.g. transformer something like this:

```diff
diff --git a/src/mrack/transformers/openstack.py b/src/mrack/transformers/openstack.py
index 1f0f1e9..854ba01 100644
--- a/src/mrack/transformers/openstack.py
+++ b/src/mrack/transformers/openstack.py
@@ -14,6 +14,7 @@
 
 """OpenStack transformer module."""
 import logging
+import os
 
 from mrack.errors import ProvisioningConfigError
 from mrack.providers.provider import STRATEGY_ABORT
@@ -41,12 +42,17 @@ class OpenStackTransformer(Transformer):
     async def init_provider(self):
         """Initialize associate provider and transformer display name."""
         self.dsp_name = "OpenStack"
+
+        os_cloud = os.environ.get('OS_CLOUD')
+        if not os_cloud:
+            os_cloud = self.config.get("profile", DEFAULT_CLOUD_PROFILE)
+
         await self._provider.init(
             image_names=self.config["images"].values(),
             networks=self.config["networks"],
             strategy=self.config.get("strategy", STRATEGY_ABORT),
             max_retry=self.config.get("max_retry", DEFAULT_ATTEMPTS),
-            cloud_profile=self.config.get("profile", DEFAULT_CLOUD_PROFILE),
+            cloud_profile=os_cloud,
             keypair=self.config["keypair"],
             pubkey=self.config["pubkey"],
```